src/components/title-blog-index/title-blog-index.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect, useState, useRef, useCallback } from "react";
+
+import { ArrowIndexIcon } from '../icons';
+import { SeeMoreTitles } from "../see-more-titles/see-more-titles";
+
+import { classnames, findTitles } from "../../helpers";
+
+import * as styles from "./title-blog-index.module.scss";
+
+const TitleBlogIndex = ({ data, refIndexTitles, disappearIndex }) => {
+  const [titleOnView, setTitleOnView] = useState('');
+  const [selectLink, setSelectLink] = useState('');
+  const [isTopArrow, setIsTopArrow] = useState(false);
+  const [isBottomArrow, setIsBottomArrow] = useState(false);
+  const subContainerRef = useRef(null);
+  const viewContainerRef = useRef(null);
+
+  useEffect(() => {
+    window.addEventListener('scroll', getActiveTitle, { passive: true });
+
+    checkOverflow();
+    const handleResize = () => checkOverflow();
```

Comment:
why this? I think the first call to checkOverflow is wrong

Reply:
Done